app/Helpers/MoneyHelper.php

```diff
@@ -40,14 +43,128 @@ public static function format($amount, Currency $currency = null)
         }
 
         $moneyCurrency = new MoneyCurrency($currency->iso);
-        $currencies = new ISOCurrencies();
-        $minorUnitAdjustment = pow(10, $currencies->subunitFor($moneyCurrency));
+        $money = new Money($amount, $moneyCurrency);
+        $numberFormatter = new \NumberFormatter(App::getLocale(), \NumberFormatter::CURRENCY);
+        $moneyFormatter = new IntlMoneyFormatter($numberFormatter, new ISOCurrencies());
 
-        $money = new Money($amount * $minorUnitAdjustment, $moneyCurrency);
+        return $moneyFormatter->format($money);
+    }
 
-        $numberFormatter = new \NumberFormatter(App::getLocale(), \NumberFormatter::CURRENCY);
-        $moneyFormatter = new IntlMoneyFormatter($numberFormatter, $currencies);
+    /**
+     * Format a monetary amount (without the currency).
+     *
+     * Amount must be an integer, in exchange format.
+     * i.e. '100' for 1,00€
+     *
+     * @param int|null $amount
+     * @param Currency|int|null $currency
+     * @return string
+     */
+    public static function formatValue($amount, $currency = null): string
+    {
+        $currency = self::getCurrency($currency);
+
+        if (! $currency) {
+            return (string) ($amount / 100);
+        }
+
+        $moneyCurrency = new MoneyCurrency($currency->iso);
+        $money = new Money($amount ?? 0, $moneyCurrency);
+        $numberFormatter = new \NumberFormatter(App::getLocale(), \NumberFormatter::PATTERN_DECIMAL);
+        $moneyFormatter = new IntlMoneyFormatter($numberFormatter, new ISOCurrencies());
 
         return $moneyFormatter->format($money);
     }
+
+    /**
+     * Format a monetary amount in exchange value with currency symbol.
+     *
+     * @param float|null $exchange
+     * @param Currency|int|null $currency
+     * @return string
+     */
+    public static function display($exchange, $currency = null): string
+    {
+        $currency = self::getCurrency($currency);
+
+        return self::format(self::formatInput($exchange, $currency), $currency);
+    }
+
+    /**
+     * Format a monetary exchange value as storable integer.
+     *
+     * @param mixed|null $exchange
+     * @param Currency|int|null $currency
+     * @return int
+     */
+    public static function formatInput($exchange, $currency): int
+    {
+        $currency = self::getCurrency($currency);
+
+        if (! $currency) {
+            return (int) ((float) $exchange * 100);
+        }
+
+        $moneyParser = new DecimalMoneyParser(new ISOCurrencies());
+        $money = $moneyParser->parse((string) $exchange, $currency->iso);
+
+        return (int) $money->getAmount();
+    }
+
+    /**
+     * Format a monetary value as exchange value.
+     *
+     * @param int|null $amount
+     * @param Currency|int|null $currency
+     * @return float
+     */
+    public static function exchangeValue($amount, $currency): float
```

Comment:
I don't understand this concept of exchange value.

Reply:
The exchange value is the amount stored as an integer, like 1234 for $12.34